packages/blocks/src/page-block/utils/container-operations.ts

```diff
@@ -550,11 +550,14 @@ export function handleKeydownAfterSelectBlocks({
 }
 export async function onModelTextUpdated(
   model: BaseBlockModel,
-  callback: (text: RichText) => void
+  callback?: (text: RichText) => void
 ) {
+  console.log('onModelTextUpdated');
```

Comment:
```suggestion
```

Reply:
will delete it in next commit, thanks for pointing it! I should be more careful